Formula/conductr-cli.rb

```diff
@@ -0,0 +1,14 @@
+class ConductrCli < Formula
+  desc "The Lightbend ConductR CLI"
+  homepage "https://conductr.lightbend.com"
+  url "https://bintray.com/lightbend/generic/download_file?file_path=conductr-cli-1.0.1-Mac_OS_X-x86_64.zip"
```

Comment:
The problem isn't necessarily it being hosted on bintray; we just don't include binary formulae in homebrew-core (more info [here](https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#we-dont-like-binary-formulae)).

If building is completely out of the question, this might be better suited for homebrew-cask or a custom tap.